src/fa/game_session.py

```diff
@@ -165,6 +173,8 @@ def _on_game_message(self, command, args):
             elif args[0] == 'Lobby':
                 # TODO: Eagerly initialize the game by hosting/joining early
                 pass
+            elif args[0] == 'Launching':
```

Comment:
If you're doing it here, I guess update_if_game_has_launched is not necessary.

Reply:
Mmh, I am not 100% sure because from what I saw there is '_on_game_message' and 'send' methods. I was thinking the 'launching' can either be when you start a hosted game or when the game is started by someone else (so you receive or send kinda...) but maybe I'm totally wrong.

I haven't tried without removing the 'update_if_game_has_launched'. Will try.

Reply:
Ok I removed the 'update_if_game_has_launched' method and it still worked.